Automattic/woocommerce-payments
# Implement BNPL providers in classic checkout flow

Fixes #6119 

#### Changes proposed in this Pull Request

Provide option of Affirm and Afterpay at checkout for eligible merchants on a classic shortcode checkout page

#### Testing instructions

Please follow the [setup instruction of the base PR](https://github.com/Automattic/woocommerce-payments/pull/6492)

For each:
- legacy UPE 
- Split UPE

1. Enable Affirm and Afterpay in payment settings
2. Add items to cart (note that Affirm is available only for purchases over 50$)
3. Make a payment using new payment method
4. Watch for any errors in console or in the front-end flow.



-------------------


**Post merge**
